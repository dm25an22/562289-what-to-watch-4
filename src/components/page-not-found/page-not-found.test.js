import React from 'react';
import NotFoundMessage from './page-not-found.jsx';
import renderer from "react-test-renderer";
import {Router} from "react-router-dom";
import {history} from "../../history";


it(`render PageNotFound component`, () => {
  const tree = renderer.create(
      <Router history={history}>
        <NotFoundMessage />
      </Router>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
