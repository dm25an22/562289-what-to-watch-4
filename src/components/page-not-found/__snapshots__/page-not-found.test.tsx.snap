// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PageNotFound component 1`] = `
Array [
  <h1
    style={
      Object {
        "fontSize": "60px",
      }
    }
  >
    Page not found 404
  </h1>,
  <a
    href="/"
    onClick={[Function]}
    style={
      Object {
        "color": "inherit",
        "fontSize": "25px",
        "listStyle": "none",
        "textDecoration": "none",
      }
    }
  >
    Go to main page
  </a>,
]
`;
